config.json

```diff
@@ -2279,6 +2279,32 @@
           "string-methods"
         ],
         "difficulty": 7
+      },
+      {
+        "slug": "secrets",
+        "name": "Secrets",
+        "uuid": "1e636fd3-6143-484e-a4fc-0ed6157fdfa1",
+        "concepts": ["anonymous-functions", "lambdas"],
```

Comment:
This is a concept exercise, so needs to follow the [concept exercise](https://github.com/exercism/docs/blob/main/building/tracks/concept-exercises.md#metadata) specifications for `config.json`.